src/util/dev/server.ts

```diff
@@ -76,11 +76,11 @@ interface NodeRequire {
 declare const __non_webpack_require__: NodeRequire;
 
 function sortBuilders(buildA: BuildConfig, buildB: BuildConfig) {
-  if (buildA.use.startsWith('@now/static-build')) {
+  if (buildA && buildA.use && buildA.use.startsWith('@now/static-build')) {
```

Comment:
If `use` is optional TypeScript should have cause this bug. Can you update the type definition of `BuildConfig` to `use?: string`